lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -358,6 +358,19 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :description => 'Sets the level of detail of log messages. Possible log levels, in increasing verbosity, are: `error`, `warn`, `info` or `debug`.'
         },
         # General
+        :active_support_custom_events_topics => {
```

Comment:
During our conversation in Slack yesterday, we talked about changing this to `active_support_custom_events_instrumentation_topics`. After taking another look at the Rails guide, I would like to propose changing "topics" to "name". 

I couldn't find a reference to the word "topics" in the documentation. Please let me know if I'm mistaken. It seems like the word "name" is used. If we use the same word, it'll be more searchable.

`active_support_custom_events_names`

Reply:
I concur. I was going after a generic pub/sub architecture with "topic" but "name" is definitely all I see in the guides. Good call. Addressed with [25a2b3b](https://github.com/newrelic/newrelic-ruby-agent/pull/1659/commits/25a2b3b04c041d371b93e721dd9e96a5c38bf525)